source/config/__init__.py

```diff
@@ -19,6 +25,18 @@
 import easeOfAccess
 import winKernel
 
+#Choices for numbers combo box in Voice settings.
+
+digitChoices=[
+	# Translators: Choice in a combo box for speaking numbers by word.
+	"Words",
+	# Translators: Choice in a combo box for speaking numbers as digits.
+	"Digits",
+	# Translators: Choice in a combo box for speaking numbers as double digits.
+	"Double Digits",
+	# Translators: Choice in a combo box for speaking numbers as triple digits.
+	"Triple Digits"]
+
 def validateConfig(configObj,validator,validationResult=None,keyList=None):
 	"""
 	@deprecated: Add-ons which need this should provide their own implementation.

```

Comment:
Hmm. Yeah, I guess config gets loaded before translation is initialised. 
The config module has never included any UI code, and translation sort 
of counts as UI. If this is used somewhere other than gui as well (I 
haven't read the code yet), put it in speech maybe. If it's just gui, 
probably just put it in settingsDialogs.